Lib/fontTools/misc/xmlWriter.py

```diff
@@ -102,12 +102,12 @@ def comment(self, data):
 
 	def simpletag(self, _TAG_, *args, **kwargs):
 		attrdata = self.stringifyattrs(*args, **kwargs)
-		data = "<%s%s/>" % (_TAG_, attrdata)
+		data = "<%s%s/>" % (str(_TAG_), attrdata)
```

Comment:
Yeh, I'm not sure about the str(), but everything falls apart without them. I think it could be to do with trying to get into bytes and not accepting Unicode as is.

Reply:
what's the traceback? can you share a test font?